### PR TITLE
text: Explain investigations

### PR DESCRIPTION
**Pull request by real-or-random**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,13 +262,26 @@ Our variant of the SimplPedPop protocol then works as follows:
...
-    Otherwise, participant `i` sets the DKG output consisting of
+    Assuming the coordinator is honest and has sent a correct `sums_com_tweaked` vector,
+    participant `i` knows that some participant contributed a wrong summand to `secshare`,
+    but participant `i` does has insufficient information to single out and blame the faulty participant.
+    In this case, participant `i` can optionally ask the coordinator for a vector `partial_pubshares`:
+    ```
+    partial_pubshares[j] = (i+1)^0 * coms[j][0] + ... + (i+1)^(t-1) * coms[j][t-1]
+    ```
+    With this vector at hand, participant `i` verifies each component of `partial_secshares` individually
+    by checking for which participant `j` the equation `partial_secshare[j] * G = partial_secshares[j]` does not hold.
```
Comment (jonasnick) [inline README.md:280]:
```suggestion
    by checking for which participant `j` the equation `partial_secshares[j] * G = partial_pubshares[j]` does not hold.
```

**Review by jonasnick — COMMENTED** [review]
Concept ACK

**Comment by real-or-random** [comment]
I added a commit that decouples tweaking from the core of SimplPedPop first. I found this conceptually easier, and easier to describe and implement. This has the potential to create some merge conflicts due to the renames, but they should be easy to resolve...

**Comment by jonasnick** [comment]
>  I found this conceptually easier, and easier to describe and implement.

I had not implemented this version because my aim was to not add any computational overhead in the happy path except what is absolutely necessary for Taproot tweaking. Your version adds an extra group addition. This may count as a downside, even just from the conceptually perspective. But I can see how your version is simpler to understand and to describe. I'm fine either way.

**Comment by real-or-random** [comment]
>  Your version adds an extra group addition.

Oh, I missed that. Fixed in https://github.com/BlockstreamResearch/bip-frost-dkg/commit/210df19cfd9fc59edb657df63599fdb3cb616d40.

**Comment by jonasnick** [comment]
Nice! I still think the simplpedpop internals description needs an update now.

**Pull request merged**